scripts/format_release_notes.py

```diff
@@ -0,0 +1,25 @@
+import shutil
+import pathlib
+import argparse
+
+current_dir = (pathlib.Path(__file__).parent / "..").resolve()
+
+
+def format_release_notes(latest_version: str):
+    shutil.copy(current_dir / "website" / "releases" / "UPCOMING.md",
```

Comment:
Should probably create a variable to reduce code duplication

`releases_dir = current_dir / "website" / "releases"`